Enabling HPC-specific test run capability (#348)
getwilds/wilds-wdl-library: modules/README.md

The test workflows automatically download test data and use hardcoded settings optimized for testing.

> If a module ships a `testrun_hpc.wdl`, that file is exercised on the monthly Fred Hutch HPC test run instead of (or in addition to) `testrun.wdl`. You can run it locally too if your environment supports it (e.g. a system with GPUs and the right environment modules) — invoke it the same way as `testrun.wdl`, or pass `TARGET=hpc` to the Makefile to have it picked up automatically.

Reviewer: ```suggestion
If a module ships a `testrun_hpc.wdl`, that file is exercised on the monthly Fred Hutch HPC test. This test also exercises  `testrun.wdl` if it exists. You can run it locally too if your environment supports it (e.g. a system with GPUs and the right environment modules). To do so, invoke it the same way as `testrun.wdl`, or pass `TARGET=hpc` to the Makefile to have it picked up automatically.
```
Minor changes for clarity (if correct)

Author: Not exactly the right wording, but you're totally right that the current wording is pretty unclear. Gave it another shot in https://github.com/getwilds/wilds-wdl-library/pull/348/commits/71dbaef8aab97299d526ad96189e7e386109878f using part of your wording.
